src/main/java/org/AppState.java

```diff
@@ -0,0 +1,93 @@
+package org;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.vehicles.Vehicle;
+
+public class AppState {
+    public static final ObjectMapper mapper = new ObjectMapper()
+            .enable(SerializationFeature.INDENT_OUTPUT);
+    public static final File SAVE_FILE = new File("app-state.json");
+
+    private List<Vehicle> activeVehiclesList = new ArrayList<>();
+    private List<Vehicle> archivedVehiclesList = new ArrayList<>();
+
+    private long serviceIdCounter = 0;
+    private long vehicleIdCounter = 0;
```

Comment:
Generally accepted practice is to generate random IDs (eg. there are libaries to genereate UUID) 

and then check for ID collision - if so then retry. otherwise you got a new ID.

[googlesearch/UUSD](https://www.google.com/search?client=firefox-b-d&q=UUID)